README.md

```diff
@@ -198,6 +206,8 @@ In order to build llama.cpp you have three different options.
     zig build -Drelease-fast
     ```
 
+Don't forget to install the Python dependencies (e.g. `python -m pip install -r requirements.txt`)
```

Comment:
Not saying this shouldn't be here, but it's already mentioned above and these "requirements" aren't for llama.cpp, but rather for converting between formats (and not necessary to use llama.cpp as a lib, for quantization, or inference). If it really needs to be in here twice, it might be better to put it with the convert.py code below.